Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/lists/modular/ModularAdapter.kt

```diff
@@ -7,6 +7,7 @@ import de.rki.coronawarnapp.ui.lists.BaseAdapter
 
 abstract class ModularAdapter<VH : ModularAdapter.VH> : BaseAdapter<VH>() {
     val modules = mutableListOf<Module>()
+    val holders = mutableListOf<VH>()
```

Comment:
Don't we create a memory leak here?

I think the RecyclerView creates the viewholders on demand and we now store a reference to all viewholders that were ever created.

Reply:
I need access to the instance of the viewholder to reset the scrolling state. Do you have a better idea?

Reply:
I'll look into it.

Reply:
I added a clear after usage which would be called onPause()